src/utils/registry.ts

```diff
@@ -562,6 +562,25 @@ export class Registry {
         else
           throw new Error(`ERROR: Playwright does not support installing ${executable.name}`);
       }
+    } catch (e) {
+      if (e.code === 'ELOCKED') {
+        const rmCommand = process.platform === 'win32' ? 'rm -R' : 'rm -rf';
```

Comment:
nit: do we need `-r` there?

Reply:
just in case we have a lockfile actually inside the lock directory at some future point of time.